.scalafmt.conf

```diff
@@ -1,51 +1,93 @@
-version = 3.9.4
+# Main goals:
+# - nicer commit diffs (trailing commas, no alignment for pattern matching, force new lines)
+# - better interop with default IntelliJ IDEA setup (matching import and modifiers sorting logic)
+# - better developer experience on laptop screens (like 16' MBPs) with IntelliJ IDEA (line wraps)
 
-project.git = true
```

Comment:
why discard it? 

https://scalameta.org/scalafmt/docs/configuration.html#projectgit?

Reply:
added it back